pkgs/development/python-modules/xmlsec/default.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs, lib, fetchPypi, buildPythonPackage, pytestCheckHook,
+pkgconfig, setuptools_scm, toml, lxml, hypothesis }:
```

Comment:
```suggestion
pkgconfig, setuptools-scm, toml, lxml, hypothesis }:
```

Reply:
The attribute is named `setuptools_scm`, and `setuptools-scm` doesn't exist. Or am I missing what you are trying to say here?

Reply:
Ah nevermind, I was looking at the wrong place.

Reply:
It was added a few weeks ago and new contributions should use the - variant.